doc/source/index.rst

```diff
@@ -32,3 +32,4 @@ Contents
    configuration/index
```

Comment:
Could you move operations under configuration?

Reply:
Do you mean just changing the order of this index, or moving the entire section to be a part of configuration?

Reply:
Just change the order